ingestion_server/ingestion_server/indices/mappings/common.json

```diff
@@ -0,0 +1,96 @@
+{
+  "authority_boost": {
+    "type": "rank_feature"
+  },
+  "authority_penalty": {
+    "positive_score_impact": false,
+    "type": "rank_feature"
+  },
+  "category": {
+    "type": "keyword"
+  },
+  "created_on": {
+    "type": "date"
+  },
+  "creator": {
+    "fields": {
+      "keyword": {
+        "ignore_above": 256,
+        "type": "keyword"
+      }
+    },
+    "type": "text"
+  },
+  "description": {
+    "analyzer": "custom_english",
+    "fields": {
+      "keyword": {
+        "ignore_above": 256,
+        "type": "keyword"
+      }
+    },
+    "similarity": "boolean",
+    "type": "text"
+  },
+  "extension": {
+    "type": "keyword"
+  },
+  "id": {
```

Comment:
I wonder why we need the separate `id` field, if we already have `_id`?

Reply:
I found this in the docs: _In case sorting or aggregating on the _id field is required, it is advised to duplicate the content of the _id field into another field that has doc_values enabled_
And we use the ids of documents to determine whether we should ingest documents (by comparing last id in ES and the DB), but does comparing fall into `sorting` or `aggregating`?

Reply:
Apparently, it does:
https://github.com/WordPress/openverse-api/blob/6cc0551c639081f895d974b30cc465713d0ce360/ingestion_server/ingestion_server/indexer.py#L200